pose_engine.py

```diff
@@ -92,15 +90,17 @@ def __init__(self, model_path, mirror=False):
             raise ValueError(
                 ('Image model should have input shape [1, height, width, 3]!'
                  ' This model has {}.'.format(self._input_tensor_shape)))
-        _, self.image_height, self.image_width, self.image_depth = self.get_input_tensor_shape()
+        _, self._input_height, self._input_width, self._input_depth = self.get_input_tensor_shape()
+        self._input_type = self._interpreter.get_input_details()[0]['dtype']
+        self._inf_time = 0
 
-        # The API returns all the output tensors flattened and concatenated. We
-        # have to figure out the boundaries from the tensor shapes & sizes.
-        offset = 0
-        self._output_offsets = [0]
-        for size in self.get_all_output_tensors_sizes():
-            offset += size
-            self._output_offsets.append(int(offset))
+    def run_inference(self, input_data):
+        """Run inference using the zero copy feature from pycoral and returns inference time in ms.
+        """
+        start = time.monotonic()
+        edgetpu.run_inference(self._interpreter, input_data)
+        self._inf_time = time.monotonic() - start
+        return self._inf_time 
```

Comment:
Why not multiply it by 1000 here to have consistency with the old one (and any code anyone has made on their own)?